x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_related_list.tsx

```diff
@@ -9,11 +9,13 @@ import { i18n } from '@kbn/i18n';
 import { EuiTitle, EuiSpacer, EuiText, EuiButtonEmpty, EuiHorizontalRule } from '@elastic/eui';
 import { useSelector } from 'react-redux';
 import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
 import { CrumbInfo, formatDate, StyledBreadcrumbs, BoldCode } from './panel_content_utilities';
 import * as event from '../../../../common/endpoint/models/event';
 import { ResolverEvent, ResolverNodeStats } from '../../../../common/endpoint/types';
 import * as selectors from '../../store/selectors';
 import { useResolverDispatch } from '../use_resolver_dispatch';
+import { RelatedEventLimitWarning } from '../limit_warnings';

```

Comment:
Would you mind giving the file name the same name as the component name in this case? This helps w/ browsing the file structure. We can see the same names in files that we've been seeing in code.

Reply:
There are two limit warnings in there - you want me to break it up into two separate files?

Reply:
nah on second thought i think this is good